README.md

```diff
@@ -191,6 +191,40 @@ public void ConfigureServices(IServiceCollection services)
 
 ```
 
+## Directory Client
+A new Directory Client can be creating as follows:
+```csharp
+
+   var logggerFactory = new NullLoggerFactory();
+   var options = new AsertoDirectoryOptions("url_and_port_to_directory_service", "directory_api_key", "directory_tenant_id", false);
+   var optionsInt = Microsoft.Extensions.Options.Options.Create(options);
+   var directoryClient = new DirectoryAPIClient(optionsInt, logggerFactory);
+
+```
+you'll need to provide the directory service URL, an API key and the Tenant ID.
+The client can be configure to not use de https by providing `options.Insecure = true;`.
+
+Example call to the directory client:
+```csharp
+
+   public async Task GetObject()
+   {
+      //...
+
+      var directoryClient = new DirectoryAPIClient(optionsInt, logggerFactory);
+
+      // Get an object.
+      var getObjectResp = await directoryClient.GetObjectAsync("object_key","object_type");
+
+      // Get the identities for a user.
+      var getRelationsResp = await directoryAPI.GetRelationsAsync(subjectType: "user", subjectKey: "userID",relationName: "identifier", relationObjectType: "identity", pageSize: 10);
```

Comment:
Can we be consistent when defining method arguments? Would it be better to have all methods with named arguments?
```csharp
var getObjectResp = await directoryClient.GetObjectAsync(objectKey: "object_key", objectType: "object_type");
// Get the identities for a user.
var getRelationsResp = await directoryAPI.GetRelationsAsync(subjectType: "user", subjectKey: "userID",relationName: "identifier", relationObjectType: "identity", pageSize: 10);
```